architecture/core_concepts/overview.adoc

```diff
@@ -7,10 +7,8 @@
 
 The following topics provide high-level, architectural information on core
 concepts and objects you will encounter when using OpenShift. Many of these
-objects come from link:../../rest_api/kubernetes_v1.html[the model provided by
-Kubernetes], which is link:../../rest_api/openshift_v1.html[extended by
-OpenShift's model] to provide a more feature-rich development lifecycle
-platform.
```

Comment:
Any particular rationale behind removing these links to the REST API? They're not very overview-y but I thought a quick link to the nuts and bolts of k8s vs openshift was germane.